docs/topics/native/native-cocoapods.md

```diff
@@ -12,26 +12,29 @@ To work correctly with Xcode, you should [update your Podfile](#update-podfile-f
 Depending on your project and purposes, you can add dependencies between [a Kotlin project and a Pod library](native-cocoapods-libraries.md)
 as well as [a Kotlin Gradle project and an Xcode project](native-cocoapods-xcode.md).
 
-## Set up the environment to work with CocoaPods
+## Set up an environment to work with CocoaPods
 
 Install the [CocoaPods dependency manager](https://cocoapods.org/):
 
 ```ruby
 sudo gem install cocoapods
 ```
 
-* If you use Kotlin prior to version 1.7.0, install the [`cocoapods-generate`](https://github.com/square/cocoapods-generate)
-plugin:
-
-  ```ruby
-  sudo gem install cocoapods-generate
-  ```
-  
-  > `cocoapods-generate` couldn't be installed on Ruby 3 and later.
-  > 
-  {type="note"}
-
-* If you encounter any problems during the installation, follow the [official CocoaPods installation guide](https://guides.cocoapods.org/using/getting-started.html#getting-started).
+<procedure initial-collapse-state="collapsed" title="If you use Kotlin prior to version 1.7.0">
```

Comment:
Maybe hide the block under expandable info? Because it is needed only for "old" version.